src/core/Form/FilterInput/FilterInput.tsx

```diff
@@ -140,6 +145,7 @@ class BaseFilterInput<T> extends Component<FilterInputProps & InnerRef> {
           >
             {labelText}
           </LabelText>
+          <HintText id={hintTextId}>{hintText}</HintText>
```

Comment:
This needs to be added to the getConditionalAriaProp call below.

Reply:
This change should now be committed.